static_src/test/server/server.js

```diff
@@ -1,6 +1,62 @@
 
+// This testing server acts as a "sidecar" for a main process given as command
+// line arguments, or simply starts the testing server.
+//
+// Sidecar pattern comes from Netflix's Prana[1] but instead of providing
+// platform services, we just provide access to the testing server.
+//
+// TODO the testing server does not require any cleanup nothing is done to stop
```

Comment:
This seems more like an observation, was confused about what actually has to be done?